client/asset/btc/btc.go

```diff
@@ -2540,19 +2584,19 @@ func (btc *baseWallet) FundingCoins(ids []dex.Bytes) (asset.Coins, error) {
 	return coins, nil
 }
 
-// Unlock unlocks the ExchangeWallet. The pw supplied should be the same as the
+// unlock unlocks the ExchangeWallet. The pw supplied should be the same as the
 // password for the underlying bitcoind wallet which will also be unlocked.
-func (btc *baseWallet) Unlock(pw []byte) error {
+func (btc *baseWallet) unlock(pw []byte) error {
 	return btc.node.walletUnlock(pw)
 }
 
-// Lock locks the ExchangeWallet and the underlying bitcoind wallet.
-func (btc *baseWallet) Lock() error {
+// lock locks the ExchangeWallet and the underlying bitcoind wallet.
+func (btc *baseWallet) lock() error {
 	return btc.node.walletLock()
 }
 
-// Locked will be true if the wallet is currently locked.
-func (btc *baseWallet) Locked() bool {
+// locked will be true if the wallet is currently locked.
+func (btc *baseWallet) locked() bool {
 	return btc.node.locked()
```

Comment:
Pretty sure this was all fine in `baseWallet`.  Comes back to @JoeGruffins' comment